changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog for [`hoopl` package](http://hackage.haskell.org/package/hoopl)
 
+## 3.10.1.1 *Apr 2015*
```

Comment:
It was the date I made the change which coincided with Andreas' change date.  I will  update it to May.